### ISSUE TITLE
@clayui/css: btn-secondary border color does not have enough contrast

### ISSUE THREAD

**Issue by pat270** [liferay/clay#5352]
`btn-secondary` border color of `#cdced9` on background `#fff` has a contrast ratio of 1.56. It should be higher. Here is the spec provided by @emiliano-cicero.

![btn-secondary](https://user-images.githubusercontent.com/788266/216666064-3a81b47b-27a3-4ace-a79c-856ba833a9cc.png)
